zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -698,7 +702,9 @@ angular.module('zeppelinWebApp')
         scrollTargetPos = documentHeight;
       }
     }
-    angular.element('body').scrollTo(scrollTargetPos, {axis: 'y', interrupt: true, duration:200});
+    angular.element('body').stop();
+    angular.element('body').finish();
+    angular.element('body').scrollTo(scrollTargetPos, {axis: 'y', interrupt: true, duration:100});
```

Comment:
@Leemoonsoo avoid using `angular.element('body')` multiple times. better to have it in a `var` and reuse.